router-tests/testenv/testenv.go

```diff
@@ -53,14 +57,16 @@ import (
 )
 
 const (
-	defaultSourceName = "default"
+	defaultSourceName = "nats"
 	myNatsSourceName  = "my-nats"
+	myKafkaSourceName = "my-kafka"
```

Comment:
Should probably be changed to ProviderId for clarity

Reply:
Done